integration-tests/deployment/environment.go

```diff
@@ -150,20 +196,33 @@ func MustPeerIDFromString(s string) p2pkey.PeerID {
 // OCR config for example.
 func NodeInfo(nodeIDs []string, oc OffchainClient) (Nodes, error) {
 	var nodes []Node
-	for _, node := range nodeIDs {
+	for _, nodeID := range nodeIDs {
 		// TODO: Filter should accept multiple nodes
 		nodeChainConfigs, err := oc.ListNodeChainConfigs(context.Background(), &nodev1.ListNodeChainConfigsRequest{Filter: &nodev1.ListNodeChainConfigsRequest_Filter{
-			NodeIds: []string{node},
+			NodeIds: []string{nodeID},
 		}})
 		if err != nil {
 			return nil, err
 		}
 		selToOCRConfig := make(map[uint64]OCRConfig)
+		bootstrap := false
+		var peerID p2pkey.PeerID
+		var multiAddr string
 		for _, chainConfig := range nodeChainConfigs.ChainConfigs {
 			if chainConfig.Chain.Type == nodev1.ChainType_CHAIN_TYPE_SOLANA {
 				// Note supported for CCIP yet.
 				continue
 			}
+			// NOTE: Assume same peerID/multiAddr for all chains.
+			// Might make sense to change proto as peerID/multiAddr is 1-1 with nodeID?
+			peerID = MustPeerIDFromString(chainConfig.Ocr2Config.P2PKeyBundle.PeerId)
+			multiAddr = chainConfig.Ocr2Config.Multiaddr
```

Comment:
The multiaddr is only populated for bootstrap

Reply:
Will it cause issues if it was set for non bootstrap nodes?

Reply:
I think leaving it blank for non-bootstraps is ok for now. @asoliman92 no don't think it will